### PR TITLE
Adding how to see the full indicator fields list

### PR DESCRIPTION
**Pull request by roysagi**
## Description
Adding how to see the full indicator fields list

## Screenshots
maybe we should add this image to the section as well?
image

### CODE REVIEW

File: docs/integrations/feeds.md [hunk 1]
```diff
@@ -154,7 +154,12 @@ Let's review the object key and values.
         URL = "URL"
     ```
 * `"rawJSON"` - _required_. This dictionary should contain the `"value"` and `"type"` fields as well as any other unmodified data returned from the feed source about an indicator.
-* `"fields"` - _optional_. A dictionary that maps values to existing indicator fields defined in Cortex XSOAR where the key is the `cliname` of an indicator field.
+* `"fields"` - _optional_. A dictionary that maps values to existing indicator fields defined in Cortex XSOAR where the key is the `cliname` of an indicator field. 
+   To see the full list of possible fields:
+   * Go to the `Settings` section in Cortex XSOAR.
+   * Click the `ADVANCED` tab. 
+   * Go to the `Fields` section and filter `indicator` fields.
+   * To inspect a specific field - tick the box near the field name and click `Edit.
```
Comment (avidan-H) [inline docs/integrations/feeds.md:162]: Mention that the `cliname` is the `machinename` when inspecting the indicator field in the UI.

File: docs/integrations/feeds.md [hunk 2]
```diff
@@ -154,7 +154,12 @@ Let's review the object key and values.
...
 * `"score"` - _optional_. The reputation score to assign to the indicator object, scores range from 0 to 3 where 0 - None, 1 - Good, 2 - Suspicious, and 3 - Bad. Assign a value only if you wish to explicitly assign a score to an indicator in your code. Typically, indicator reputation is set at a Feed Integration configuration level by setting the `Indicator Reputation` parameter when configuring an instance.
```
Comment (avidan-H) [inline docs/integrations/feeds.md:163]: Also, please remove `score` as this seems to be removed from top-level feed indicator mapping on the server.

Comment (roysagi) [reply docs/integrations/feeds.md:163]: done

**Comment by glicht** [comment]
@roysagi status?

**Comment by roysagi** [comment]
@glicht - Added the image right now, merging as soon as it passes build.

**Pull request merged**